Unable to build on Raspberry Pi due to new mise build dependency

### Prerequisites

- [x] I am using the latest release of System Bridge.
- [x] I have read the README and relevant documentation.

### System Bridge version

5.5.0

### Installation method

Built from source (make build)

### Installation method details

Build from source on Raspberry Pi and install binary manually

### Operating system

Linux

### OS details

Raspbian GNU/Linux 13 (trixie) on armhf

### Describe the bug

In prior releases, I could build system-bridge for my Raspberry Pi from source with no issues and park the binary where systemd knew where to look.

This new version seems to have added the [mise](https://mise.jdx.dev/) tool as part of the build, which is not available as a package for the Pi's version of Debian, which means that I need to build that from scratch, which means that I need to build and install the rust toolchain, as well.

By the time I've rebuilt and reinstalled all the new build dependencies, my device runs out of HDD space (it's a small embedded system) before I get to building system-bridge.

If possible, can you either revert the mise addition OR generate an armhf .deb file as part of the build process?

### Steps to reproduce

1. On a Raspberry Pi, download and unpack the source and run "make build".

2. The build aborts when it can't find the mise command.

### Expected behavior

I should be able to run "make build" on the Raspberry Pi without having to build the entire Rust toolchain.

### Actual behavior

Console log:

````
cjkarr@pi:~/Documents/Projects/system-bridge $ make build
rm -f system-bridge 2>/dev/null
rm -f system-bridge-linux 2>/dev/null
rm -rf web-client/dist 2>/dev/null
Generating Zod schemas from Go types...
make: mise: No such file or directory
make: *** [Makefile:44: generate_schemas] Error 127
````

### Relevant logs/output

```bash

```

### Additional context / screenshots

_No response_

--- a/Makefile
+++ b/Makefile
@@ -1,5 +1,7 @@
-# OS detection
-MISE=mise exec --
+# Use mise if available; fall back to direct tool invocation for environments
+# without mise (e.g. Raspberry Pi source builds). Override with MISE= to force
+# direct invocation even when mise is installed.
+MISE := $(shell command -v mise >/dev/null 2>&1 && printf 'mise exec --')
 
 ifeq ($(OS),Windows_NT)
 	EXE=.exe
@@ -106,10 +108,10 @@ endif
 build_tui:
 	@echo "Building TUI..."
 ifeq ($(OS),Windows_NT)
-	cd tui && bun install --frozen-lockfile && bun build src/index.ts --compile --outfile ../system-bridge-tui.exe
+	cd tui && $(MISE) bun install --frozen-lockfile && $(MISE) bun build src/index.ts --compile --outfile ../system-bridge-tui.exe
 	@echo "TUI built: system-bridge-tui.exe"
 else
-	cd tui && bun install --frozen-lockfile && bun build src/index.ts --compile --outfile ../system-bridge-tui
+	cd tui && $(MISE) bun install --frozen-lockfile && $(MISE) bun build src/index.ts --compile --outfile ../system-bridge-tui
 	@echo "TUI built: system-bridge-tui"
 endif
 
@@ -149,7 +151,7 @@ install: build
 
 run: build
 	@echo "Starting web client and backend in parallel..."
-	cd web-client && $(MISE) pnpm exec concurrently -n "web,backend" -c "blue,green" "mise exec -- pnpm dev" "../$(OUT) backend"
+	cd web-client && $(MISE) pnpm exec concurrently -n "web,backend" -c "blue,green" "$(MISE) pnpm dev" "../$(OUT) backend"
 
 run-web-client:
 	cd web-client && $(MISE) pnpm dev

--- a/README.md
+++ b/README.md
@@ -34,11 +34,16 @@ See [installation documentation](https://system-bridge.timmo.dev/docs/install).
 
 ## Development Setup
 
-1. Install `mise`
+1. Install [`mise`](https://mise.jdx.dev/installing-mise.html) (recommended)
 1. Run `mise install` in the repo root to install the pinned Go, Node,
    and pnpm toolchains.
 
 `mise.toml` is the source of truth for local and CI runtimes.
+
+> **Building without mise:** If you cannot install `mise` (e.g. Raspberry Pi
+> armhf), ensure compatible versions of `go`, `node`, `pnpm`, and `bun` are on
+> your PATH. The Makefile auto-detects `mise` and falls back to invoking tools
+> directly when it is not available.
 
 ## Build and Install
 

